get-mods.sh

```diff
@@ -22,7 +22,9 @@ get_special () {
 				REF="$(get_tag)"
 				;;
 			*)
-				if [ -n "$BRANCH" ] && git -C dockerweb2-temp show -s --oneline "$BRANCH"; then
+				if [ "$BRANCH" = master ] && [[ "$2" == icinga-php/* ]]; then
```

Comment:
What does `[[ "$2" == icinga-php/* ]]` do, why is it needed...

Reply:
Match the destination directory against a wildcard? Because I only want to override the branch name on a specific combination?